### PR TITLE
Fix bug with with unifying the untrated values

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Added unify for untreated values
Related JIRA issue: GDR-1933

## Why was it changed?
To unify untreated values

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Comment by bczech** [comment]
FYI @darsoo

File: R/create_SE.R [hunk 1]
```diff
@@ -76,6 +76,10 @@ create_SE <- function(df_,
     )
     df_[single_agent_idx, drug2_var] <- untreated_tag[1]
   }
+  
+  df_ <- as.data.frame(lapply(df_, function(x) {
```
Comment (marcin-kam) [inline R/create_SE.R:80]: Shouldn't it be data table as we are getting rid of data frames?

File: DESCRIPTION [hunk 2]
```diff
@@ -44,7 +44,8 @@ Suggests:
     pkgbuild,
     qs,
     testthat,
-    yaml
+    yaml,
+    gDRinternalData
```
[the remark this answers is not in the document]
Comment (bczech) [reply DESCRIPTION:48]: On Friday, CI failed due to a lack of this in the DESCRIPTION. I think it was some random error generated, I will remove it from DESCRIPTION and check today again

File: R/fit_SE.combinations.R [hunk 3]
```diff
@@ -158,7 +158,7 @@ fit_SE.combinations <- function(se,
       # secondary one
       
       col_fittings <- fit_combo_cotreatments(
-        avg_combo,
+        avg_combo[normalization_type == metric],
```
Comment (marcin-kam) [inline R/fit_SE.combinations.R:161]: Maybe extract it to variable as it's called three times?

**Review by marcin-kam — APPROVED** [review]

**Pull request merged**